tutorials/3d/ragdoll_system.rst

```diff
@@ -0,0 +1,92 @@
+.. _doc_ragdoll_system:
+
+Ragdoll system
+===============
+
+Introduction
+------------
+
+A ragdoll is the use of physics body's simulation to control the bone of a mesh. It is used for death animations in many games and other kind of physics-based animation. Godot has a built-in ragdoll system. We will be using the Platformer3D demo to demonstrate the set up of a ragdoll.
+
+.. note:: You can download the Platformer3D demo on `GitHub <https://github.com/godotengine/godot-demo-projects/tree/master/3d/platformer>` or `the Asset Library <https://godotengine.org/asset-library/asset/125>`_.
+
+Setting up the ragdoll
+--------------------
+
+Create physical bones
+~~~~~~~~~~~~~~~~~~~~~
+
+Similar to many godot's features, there is a node for setting up a ragdoll. You will be using a :ref:`PhysicalBone <class_PhysicalBone>` node. To simplify the set up, you can create the required physical bones with the feature "Create physical skeleton" in the skeleton node.
+
+In the demo, select the skeleton node (in the player scene of Robi the robot). You will notice that a skeleton button appears on the top bar menu:
```

Comment:
Avoid parens. There's rarely a need for them, and they break the flow of your sentence. 
E.g. "Open the demo in Godot, then open the player scene: `Robi.tscn`. In the node tab, select the skeleton node.